spyder-custom-layout/.pre-commit-config.yaml

```diff
@@ -0,0 +1,13 @@
+# ----------------------------------------------------------------------------
```

Comment:
Please remove this file. We shouldn't force plugin developers to format everything with black.

Besides, using pre-commit is very slow.